### PR TITLE
Add listC and listP commands

### LINKED ISSUE #91: Add listC and listP commands
As a university student, I want to be able to list out contacts and projects, so that I can find the contact or project I am looking for.

### PR DESCRIPTION
**Pull request by samuelfangjw**
listC displays list of contacts, listP displays list of projects

closes #91

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ListProjectsCommand.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
```
Comment (vevek) [inline src/main/java/seedu/address/logic/commands/ListProjectsCommand.java]: Should the predicate being imported be PROJECTS instead?

File: src/main/java/seedu/address/logic/commands/ListProjectsCommand.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    public static final String COMMAND_WORD = "listP";
+
+    public static final String MESSAGE_SUCCESS = "Listed all projects";
+
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```
Comment (vevek) [inline src/main/java/seedu/address/logic/commands/ListProjectsCommand.java]: Here too.

**Review by vevek — CHANGES_REQUESTED** [review]
Just the 2 comments to review.

File: src/main/java/seedu/address/logic/commands/ListProjectsCommand.java (hunk 1 above)
Comment (samuelfangjw) [reply src/main/java/seedu/address/logic/commands/ListProjectsCommand.java]: yup good spot.

**Review by vevek — APPROVED** [review]
LGTM

**Pull request merged**